docs/ipfs-protocol-handler-support-tests.html

```diff
@@ -0,0 +1,131 @@
+ <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link rel="stylesheet" href="//unpkg.com/tachyons@4.11.1/css/tachyons.min.css" type="text/css">
+  <link rel="stylesheet" href="//unpkg.com/ipfs-css@0.13.1/ipfs.css" type="text/css">
+  </head>
+
+<body class='sans-serif bg-white'>
+  <header class='pv3 ph2 ph3-l bg-navy cf'>
+    <a href="https://ipfs.io/" title='ipfs.io'>
+      <img src="https://ipfs.io/images/ipfs-logo.svg" class='v-mid' style='height:50px' />
+    </a>
+    <h1 class='aqua fw2 montserrat dib ma0 pv2 ph1 v-mid fr f3 lh-copy'><code></code> protocol handler smoke-tests</h1>
+  </header>
+  <section class='pv4 pl2 pl5-l'>
+
+    <p class="charcoal-muted">
+      Below are checks for protocol handler support defined in <a class='charcoal-muted hover-teal link underline' href="https://github.com/ipfs/in-web-browsers/blob/master/ADDRESSING.md">IPFS Addressing Spec</a>.<br/>
+      Source: <a class='charcoal-muted hover-teal link underline' href="https://github.com/ipfs/in-web-browsers">ipfs/in-web-browsers</a>
+    </p>
+
+    <div class='pt4'>
+      <div class='db pb1 w-100 fw2 tracked ttu f3 teal-muted link'>
+        A HREF
+      </div>
+      <p>Clicking on each link should work:</p>
```

Comment:
Can you be more explicit about what "work" means? eg "should result in HTTP 200 OK with expected content loading. 

Also, should provide the HTTP link for verification for each, so a tester can ensure the content matches.